src/controller/CommandController.java

```diff
@@ -68,29 +68,29 @@ public int removeField(String className, String fieldName) {
 		return result;
 	}
 
-	public int addMethod(String className, String methodName) {
-		int result = getModel().addMethods(className, methodName);
+	public int addMethod(String className, String methodName, String returnType, String params) {

```

Comment:
Same as field, may want to swap order of method input to match typical programming format. Design choice.